Sources/Tokenizer/Styles/PropertyHelper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  StylingHelper.swift
+//  Differentiator-iOS
+//
+//  Created by Matouš Hýbl on 05/03/2018.
+//
+
+import Foundation
+
+public struct PropertyHelper {
+    
+    public static func deserializeSupportedProperties(properties: [PropertyDescription], in element: SWXMLHash.XMLElement) throws -> [Property] {
+        var result = [] as [Property]
+        for (attributeName, attribute) in element.allAttributes {
+            guard let propertyDescription = properties.first(where: { $0.matches(attributeName: attributeName) }) else {
+                continue
+            }
+            //            guard
```

Comment:
Do you think we could get rid of this dead code, or is it still relevant?

Reply:
I'll investigate on what does it do and whether it was moved to somewhere else, but I'll keep it here until the big refactoring.